contracts/BaalSummoner.sol

```diff
@@ -4,47 +4,63 @@ pragma solidity 0.8.7;
 import "@gnosis.pm/zodiac/contracts/factory/ModuleProxyFactory.sol";
 import "@gnosis.pm/safe-contracts/contracts/proxies/GnosisSafeProxyFactory.sol";
 import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 
 import "./Baal.sol";
 
-contract BaalSummoner is ModuleProxyFactory {
```

Comment:
remove ModuleProxyFactory because we are no longer using createProxy and instead using the deployed library. THis is why we have to pass at least some call function in the moduleProxyFactory.deployModule(...)